src/frontend/citizen-portal/src/app/components/landing/landing.component.html

```diff
@@ -87,7 +87,12 @@ <h4>Did you know?</h4>
       <div class="flex-2-col info">
         <h4>Useful links</h4>
 
-        <mat-expansion-panel class="pt-0">
+        <button mat-button class="link-button pt-0 pb-3" [routerLink]="['/dispute/find']">
+          Manage or update my dispute
```

Comment:
To help with our huge tech debt for this project, whenever I see hardcoded text in the UI, I've been replacing it with the better practice of using the i18n files. Can we do that here? Place this hard-coded text in the en.json and fr.json language files and use:
`{{ "labels.manage_update_dispute" | translate }}`

Reply:
Added label